server/graphql/v2/input/PaymentMethodReferenceInput.js

```diff
@@ -0,0 +1,15 @@
+import { GraphQLInputObjectType, GraphQLInt, GraphQLString } from 'graphql';
+
+export const PaymentMethodReferenceInput = new GraphQLInputObjectType({
+  name: 'PaymentMethodReferenceInput',
+  fields: () => ({
+    uuid: {
+      type: GraphQLString,
+      description: 'The uuid string assigned to the payment method',
+    },
+    CollectiveId: {
```

Comment:
I don't see the use case for this parameter.

Reply:
The UUID? It's used in the `updateOrder` mutation where we find the payment method via uuid